build/images/base-windows/README.md

```diff
@@ -0,0 +1,22 @@
+# images/base-windows
+
+This Docker image includes download libraries for building windows image.
+
+If you need to build a new version of the image and push it to Dockerhub, you
+can run the following:
+
+```bash
+cd build/images/base-windows
+WIN_BUILD_TAG= echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION| md5sum| head -c 10
+docker build -t antrea/base-windows:<WIN_BUILD_TAG> --network host .
+docker push antrea/base-windows:<WIN_BUILD_TAG>
+```
```

Comment:
The script cannot be executed directly.
1. The second line seems assigning the tag to a variable called WIN_BUILD_TAG but it actually didn't
2. The 3rd and 4th lines seems using the variable but need manual replacement, why doesn't it use the variable directly?
3. Executing the script need some enviroment variables: GO_VERSION, CNI_BINARIES_VERSION, NANOSERVER_VERSION, can we add `GO_VERSION=$(head -n1 ../deps/go-version)` to the script?